grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java

```diff
@@ -115,26 +118,26 @@ private void refreshGraph() {
     }
 
     void handleMessage(Json json) {
-        switch (json.at(REST.RemoteShell.ACTION).asString()) {
-            case REST.RemoteShell.ACTION_QUERY:
+        switch (json.at(ACTION).asString()) {
+            case ACTION_QUERY:
                 receiveQuery(json);
                 break;
-            case REST.RemoteShell.ACTION_END:
+            case ACTION_END:
                 executeQuery();
                 break;
-            case REST.RemoteShell.ACTION_QUERY_ABORT:
+            case ACTION_QUERY_ABORT:
                 abortQuery();
                 break;
-            case REST.RemoteShell.ACTION_COMMIT:
+            case ACTION_COMMIT:
                 commit();
                 break;
-            case REST.RemoteShell.ACTION_ROLLBACK:
+            case ACTION_ROLLBACK:
                 rollback();
                 break;
-            case REST.RemoteShell.ACTION_DISPLAY:
+            case ACTION_DISPLAY:
                 setDisplayOptions(json);
                 break;
-            case REST.RemoteShell.ACTION_PING:
+            case ACTION_PING:
```

Comment:
The pings only exist to keep the connection open, so it's meant to be ignored (although I could make this clearer in comments).